docroot/profiles/favrskov/modules/features/views/views_job/views_job.info

```diff
@@ -2,7 +2,7 @@ name = Views Job
 description = Presents Job Importer search and Job list.
 core = 7.x
 package = Views
-version = 7.x-1.0
+version = 7.x-1.2
```

Comment:
Why we jump directly to 1.2 instead of 1.1 ?

Reply:
The feature was overriden, so I recreated it to version 1.1. Then I applied modifications to a view, so I recreated it again to version 1.2.